jaxb/pom.xml

```diff
@@ -128,57 +128,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
-                <artifactId>maven-javadoc-plugin</artifactId>
-                <version>2.10.4</version>
-                <executions>
-                    <execution>
-                        <id>attach-javadocs</id>
-                        <goals>
-                            <goal>jar</goal>
-                        </goals>
-                    </execution>
-                </executions>
-                <configuration>
-                    <excludePackageNames>no.digipost.signature.api.xml.thirdparty.xmldsig</excludePackageNames>
-                    <additionalparam>-Xdoclint:-missing</additionalparam>
-                </configuration>
-            </plugin>
-            <plugin>
-                <artifactId>maven-source-plugin</artifactId>
```

Comment:
Denne fjernes også fordi den finnes i super-pomen?

Reply:
Det er p.g.a. det trikset som er beskrevet her https://github.com/digipost/digipost-open-super-pom#build-source-and-javadoc-artifacts